### PR TITLE
Detect WebGL support on BrowserInfo

### PR DESCRIPTION
**Pull request by philippjfr**
Detects webgl support, e.g. so we can toggle webgl rendering automatically in HoloViews.

### CODE REVIEW

File: panel/models/browser.ts [hunk 1]
```diff
@@ -24,6 +24,15 @@ export class BrowserInfoView extends View {
...
       this.model.timezone_offset = timezone_offset
     }
+    try {
+      var canvas = document.createElement('canvas');
+      this.model.webgl = !!(
+        window.WebGLRenderingContext &&
+          (canvas.getContext('webgl') || canvas.getContext('experimental-webgl'))
+      );
+    } catch (e) {
+      this.model.webgl
```
Comment (philippjfr) [inline panel/models/browser.ts:34]:
```suggestion
      this.model.webgl = false
```

File: panel/models/browser.ts [hunk 2]
```diff
@@ -24,6 +24,15 @@ export class BrowserInfoView extends View {
     if (timezone_offset != null) {
       this.model.timezone_offset = timezone_offset
     }
+    try {
+      var canvas = document.createElement('canvas');
+      this.model.webgl = !!(
+        window.WebGLRenderingContext &&
+          (canvas.getContext('webgl') || canvas.getContext('experimental-webgl'))
```
Comment (philippjfr) [inline panel/models/browser.ts:31]:
```suggestion
      var canvas = document.createElement("canvas");
      this.model.webgl = !!(
        window.WebGLRenderingContext &&
          (canvas.getContext("webgl") || canvas.getContext("experimental-webgl"))
```

File: panel/pane/holoviews.py [hunk 3]
```diff
@@ -505,6 +505,8 @@ def _render(self, doc, comm, root):
                 Store.current_backend = loaded_backend
             backend = self.backend or Store.current_backend
             renderer = Store.renderers[backend]
+        if state.browser_info and 'webgl' in renderer.param:
+            renderer.webgl = state.browser_info.webgl
```
Comment (hoxbro) [inline panel/pane/holoviews.py:509]: I don't think it is Panel job to do this.

**Review by hoxbro — COMMENTED** [review]
An electron bug caused the issue with Jupyter Desktop. See the comments at https://github.com/bokeh/bokeh/issues/13665. It was fixed by deleting a specific folder. 

I don't know if this would solve the problem more than hide it. I could be wrong.

File: panel/pane/holoviews.py (hunk 3 above)
Comment (philippjfr) [reply panel/pane/holoviews.py:509]: Can see arguments both ways. I'd say this is the best place to do it though.

Comment (hoxbro) [inline panel/pane/holoviews.py:509]:
```suggestion
```

**Review by hoxbro — APPROVED** [review]

**Pull request merged**